README.md

```diff
@@ -1,2 +1,60 @@
-# Musical-Fountain
-Srishti 2019
+
+#MUSICAL FOUNTAIN  
+
+![Image 1](https://github.com/kshitij2116/Musical-Fountain/blob/master/Images%20and%20Videos/Images/IMG_2318.JPG)
+![Image 2](https://github.com/kshitij2116/Musical-Fountain/blob/master/Images%20and%20Videos/Images/IMG_2327.JPG)  
+
+##ABSTRACT
```

Comment:
In all the heading there should be space after ##. That's why you are not getting the desired output.